src/components/LoginPage.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+const LoginPage = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const history = useHistory();
+
+  const handleLogin = (event) => {
+    event.preventDefault();
+
```

Comment:
Hardcoding the username and password in the component is not a good practice. It would be better to move this to a configuration file or environment variables.